### DISCUSSION TITLE
Comparing scVelo and CytoTrace results

### DISCUSSION THREAD

**Discussion by gundalav** [theislab/scvelo#565] [Q&A]
I have the following scRNaseq analysis done with **[scVelo][1]** and [**CytoTrace**][2].

<img width="1216" alt="Screen Shot 2021-07-29 at 17 17 09" src="https://user-images.githubusercontent.com/8143832/127459175-010776c7-ae54-4184-85e8-1d2ce1128f5a.png">

My understanding is as follows:

 1. scVelo arrows lead from less differentiated cell X  (more stem cell) to more differentiated cell Y (e.g. granulocyte)
 2. CytoTrace brighter color is more differentiated (e.g. granulocyte) to darker color less differentiated (e.g.  more esteem)

If so, the two results I have above are totally opposite. 
Is my interpretation correct? If so, why did it give the opposite result?
Which one should we trust: scVelo or CytoTrace?

  [1]: https://scvelo.readthedocs.io
  [2]: https://cytotrace.stanford.edu/

**Comment by WeilerP** [comment]
@gundalav, given the limited info, I can only guess to why the results differ. If their is a problem with the inference in scVelo it is most likely due to uninformative phase portraits (see e.g. #216 or #462). CytoTRACE will only work if the base assumption is met, i.e. if less mature cells express more genes.
Concluding, you cannot make a general statement about trusting one method more than another, IMHO. It always depends on the dataset you are using whether or not the method may work.
